### PR TITLE
Remove Fedora/Tophat

### PR DESCRIPTION
**Pull request by jkadamczyk**
## What changed (plus any additional context for devs)

* Removes Fedora feature from the app based on Pull requests that introduced it (#3149 and #3185)
* Merging that requires some QA and afterwards removing CI flows that backed this up

## What to test
* The app should just work and load bundles properly, this feature was replacing bundle with a one downloaded with codepush after scanning a special QR code

### CODE REVIEW

**Review by skylarbarrera — APPROVED** [review]
🚢

File: src/App.js [hunk 1]
```diff
@@ -396,9 +370,7 @@ function Root() {
 }
 
 const RootWithSentry = Sentry.wrap(Root);
-const RootWithCodePush = codePush({
-  checkFrequency: codePush.CheckFrequency.MANUAL,
-})(RootWithSentry);
+const RootWithCodePush = codePush(RootWithSentry);
```
Comment (estrattonbailey) [inline src/App.js:373]: Do we still need `codePush` here?

**Review by estrattonbailey — COMMENTED** [review]
LGTM

File: src/App.js (hunk 1 above)
Comment (jkadamczyk) [reply src/App.js:373]: This feature used codepush, but we use codepush separately as well. We haven't used that in a long time so generally might be worth checking if we actually still need it, but yeah it is also configured for OTA updates

**Review by brunobar79 — APPROVED** [review]

**Pull request merged**